buildlib/az-helpers.sh

```diff
@@ -189,7 +189,7 @@ try_load_cuda_env() {
     [ "${num_gpus}" -gt 0 ] || return 0
 
     # Check cuda env module
-    az_module_load dev/cuda12.2.2 || return 0
+    az_module_load dev/cuda12.3.2 || return 0
```

Comment:
can we use the newer cuda 12.5?

Reply:
Sure. Please also note the updated GDRCopy module.